tests/dummy/app/templates/docs/sprites.md

```diff
@@ -0,0 +1,68 @@
+# Sprites
+
+## What is a sprite?
+
+A sprite is an html element that can animate by using its starting and ending positions. The term "sprite" comes from a long tradition of two-dimensional graphics. In ember-animated, sprites carry additional data about themselves. This extra information is typically the initial and/or the final location of the sprite. Sprites can animate from their starting position to their final position and vice versa. 
+
+The initial state of a sprite is dimensions of the location where a sprite starts at. The dimensions of the location that the sprite ends at is the final state of that sprite. There are five sprite categories: `insertedSprites`, `keptSprites`, `removedSprites`, `sentSprites`, and `receivedSprites`. The first three categories are outlined here: 
+
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;}
+.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;}
+.tg .tg-eh2d{background-color:#ffffff;border-color:inherit;vertical-align:top}
+.tg .tg-47u2{font-weight:bold;background-color:#ffffff;border-color:inherit;vertical-align:top;text-align:left}
+.tg .tg-7g6k{font-weight:bold;background-color:#ffffff;border-color:inherit;text-align:center;vertical-align:top}
+</style>
+<table class="tg">
+  <tr>
+    <th class="tg-47u2">Name</th>
+    <th class="tg-7g6k">Initial State</th>
+    <th class="tg-47u2">Final State</th>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Inserted</td>
+    <td class="tg-eh2d">No</td>
+    <td class="tg-eh2d">Yes</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Kept</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">Yes</td>
+  </tr>
+  <tr>
+    <td class="tg-eh2d">Removed</td>
+    <td class="tg-eh2d">Yes</td>
+    <td class="tg-eh2d">No</td>
+  </tr>
+</table>
+
```

Comment:
This table needs a little more explanation. Maybe I don't understand the significance of the "initial" versus "final" states without those being explicitly described.